### ISSUE TITLE
[Feature]: Bik video to mp4 convertion using ffmpeg

### ISSUE THREAD

**Issue by daervik** [Xaymar/Hellextractor#24]
### Describe the Feature

If this is out of scope for the Hellextractor, please remove this request.

I did a very simple test with a bik file (b2b8b34553f957cb.bik) from the extraction using a very basic ffmpeg command line command using ffmpeg essentials build and command ".\ffmpeg.exe -i b2b8b34553f957cb.bik b2b8b34553f957cb.mp4", which resulted in an encoded mp4 with the video and sound from the bik file.

If it is of interest to make the exports from Hellextractor as user friendly as possible it could be interesting to implement ffmpeg into the tool, possibly with a video file format flag (mp4, mkv etc).

Some bik video files are without sound, so not sure if those have sound separated out due to them being played back separately in engine.

**Comment by Xaymar** [comment]
Yeah, this is a bit out of scope for now. I could implement it, but considering that precalculating the size of the converted file is practically impossible, it'd have to redo the conversion every single export.

**Comment by daervik** [comment]
Maybe I'm misunderstanding, but what purpose does it serve to precalculate the size of the converted file?

**Comment by Xaymar** [comment]
Not having to export it again, and being able to rename from old names to new names.
